tests/pages/nodes/node/HealthTab-cy.js

```diff
@@ -36,15 +42,15 @@ describe('Node Health Tab [0fa]', function () {
       cy.get('@filterHealth').click();
       cy.get('.dropdown').find('li').contains('Healthy').click();
       cy.get('.page-content td .text-success').should(function ($row) {
-        expect($row.length).to.equal(18);
+        expect($row.length).to.be.at.least(5);
```

Comment:
It's a good number :smile:. Ok, seriously: There are certain health components we can rely on being there and having at least 5 tests that there are some without having to test the exact number. I can add a comment for this if this deserves an explanation